### ISSUE TITLE
playground-docs container restarting

### ISSUE THREAD

**Issue by bitkarrot** [PLEBNET-PLAYGROUND/plebnet-playground-docker#136]
On `docker-compose up -d`, playground-docs container restarting due to error on master branch, commit 7020d60ff27aec54876908291a0f94d36e01c6af

Running on OSX, Ventura 13.0, arm arch

```
Error: MkDocs encountered an error parsing the configuration file: while constructing a Python object
cannot find module 'materialx.emoji' (No module named 'materialx')
  in "/docs/mkdocs.yml", line 143, column 20
```

**Comment by RandyMcMillan** [comment]
i am taking a look at this...

**Comment by RandyMcMillan** [comment]
fixed with commit: 21bbd45532c9266fa2f2db46dd2cbb28f715a8af
please test to confirm... thanks! :)

**Comment by bitkarrot** [comment]
appears merged, closing
